dytype/dytype.scm

```diff
@@ -110,9 +112,13 @@
 (define prelude
   '(
     (+ . (* . 1))
+    (modulo . 3)
+    (number->string . 2)
     (cons . 3)
     (list . (* . 1))
+    (iota . 4)
```

Comment:
I'm not familiar with `iota`. I guess it's your enumerator?

Reply:
Yeah, it's a lot like python2's `range` but with the arguments all over the place.

Reply:
According to the docs the name is from APL, haha!